# Filtered bottom flow in bodyframe.

float32 sumx				# Integrated bodyframe x flow in meters
float32 sumy				# Integrated bodyframe y flow in meters

float32 vx 				# Flow bodyframe x speed, m/s
float32 vy				# Flow bodyframe y Speed, m/s
